hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/RatisPipelineProvider.java

```diff
@@ -157,12 +157,11 @@ public Pipeline create(ReplicationFactor factor) throws IOException {
         new CreatePipelineCommand(pipeline.getId(), pipeline.getType(),
             factor, dns);
 
-    dns.stream().forEach(node -> {
-      final CommandForDatanode datanodeCommand =
-          new CommandForDatanode<>(node.getUuid(), createCommand);
+    dns.forEach(node -> {
       LOG.info("Send pipeline:{} create command to datanode {}",
```

Comment:
Since we explicitly call group.add this log message seems out of date, maybe it should say: "Adding datanode {} to pipeline {}"

Reply:
Thanks for the review @swagle 
This is SCM part and we are not explicitly calling groupAdd from here. We are queuing datanode command to create the pipeline.
I will rephrase it to `Sending CreatePipelineCommand for pipeline:{} to datanode: {}`